TFT/src/User/API/Printing.c

```diff
@@ -452,11 +486,15 @@ void printAbort(void)
 
   switch (infoFile.source)
   {
+    case REMOTE_HOST:  // nothing to do
+      loopDetected = false;
+      return;
+
     case BOARD_SD_REMOTE:
     case BOARD_SD:
-      //infoHost.printing = false;  // Not so fast! Let Marlin tell that he's done!
+      //infoHost.printing = false;  // not so fast! Let Marlin tell that he's done!
```

Comment:
Sentences in English always start with capital letter.

Reply:
I simply use all the same comment style (lower case) in the files

Reply:
It's basic grammar, sentences always have to start with capital letter. It's really not debatable.